lib/storage/data/file/utils.js

```diff
@@ -0,0 +1,19 @@
+const posixFadvise = require('fcntl');
+
+/**
+ * Release free cached pages associated with a file
+ *
+ * @param {String} filePath - absolute path of the associated file
+ * @param {Int} fd - file descriptor of the associated file
+ * @param {werelogs.RequestLogger} log - logging object
+ * @return {undefined}
+ */
+function releasePageCacheSync(filePath, fd, log) {
+    const ret = posixFadvise(fd, 0, 0, 4);
```

Comment:
`posixFadvise` was previously `posixFadvise(fd, 0, size, 4);`. We don't need the set the size param?

Reply:
`man posix_fadvise` says that len 0 affects bytes until the end of the file, I think in this context it's good since we release the cache memory for the whole file.